README.md

```diff
@@ -1,6 +1,31 @@
 # Eclipse Che devfile registry
 
-This repository holds ready-to-use Devfiles for different languages and technologies.
+This repository holds meta.yaml files that refer to the samples with Devfiles.
+
+Here is the expected format of a `meta.yaml`:
+
+```yaml
+---
+displayName: Python
+description: Python Stack with Python 3.8
+tags: ["Community", "Centos", "Python", "pip"]
+icon: /images/python.svg
+links:
+  v2: https://github.com/che-samples/python-hello-world/tree/main
+```
+
+Here are all the supported values:
```

Comment:
what follows is a verbatim copy of the sample meta.yaml, not an explanation of the supported values. Copy and paste error?

Reply:
added more comments